latitudesh/lsh
# Feat: Add ability to render nested attributes to the table output

## What

This PR refactors the table renderer to allow each resource to tell how its own attributes will be rendered.

With that, we expect to be able to create custom columns for nested fields on tables, or even grouping different attributes into the same column.

This PR also renames the `format` flag to `output`. Now if you want to output as JSON, you can run `--json` or `-o json`, for example.

## How to test

For an effective test, all the responses should be tested. You can build the app and run `./lsh --help` to check the available commands (this will be helpful to also test if our `--help` command is providing the correct guidance).

Apart from errors and deletions, for every response you should see a table with the values in it. There are a few details to improve (apart from the code itself) that have been already mapped by the team, like:

- Remove columns with no values
- Better rendering for prices
- Alternative rendering for empty lists
